### ISSUE TITLE
Help: Cannot add project into new pull request

### ISSUE THREAD

**Issue by zigang93** [srggrs/assign-one-project-github-action#57]
I was able to add project into the issue. 
but not working on fork pull request.
Am I doing something wrong??

P/S: my orgs repo is private and the fork repo also private

```yaml
name: Auto PR/Issue Assign to Project(s)

on:
  issues:
    types: [opened, labeled]
  pull_request:
    types: [opened]

env:
  MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}

jobs:
  assign_one_project:
    runs-on: ubuntu-latest
    name: Assign to One Project
    steps:
    - name: Assign NEW issues to project
      uses: srggrs/assign-one-project-github-action@1.2.0
      if: github.event.action == 'opened'
      with:
        project: 'https://github.com/orgs/example/projects/5'
        column_name: 'To Do'
    
    - name: Assign NEW PR to project
      uses: srggrs/assign-one-project-github-action@1.2.0
      if: |
        github.event == 'pull_request' &&
        (
          github.event.action == 'opened'
        )
      with:
        project: 'https://github.com/orgs/example/projects/5'
        column_name: 'In Progress'
```

**Comment by srggrs** [comment]
Hi! The workflow file seems ok, the only thing is to make sure that your project are set up correctly, so the `To Do` and `In Progress` columns are there, if not or the names do not match (I think it is ~case sensitive~) the action will fail. 

Anyhow are you able to post some of the traceback from the action log? You can find it under the "Actions" menu, selecting a particular run of the action and then click on the particular job/action... you should end up with something like this:
![image](https://user-images.githubusercontent.com/34258464/101760852-88d38f80-3b2f-11eb-8f59-b1dddd76c01f.png)

**Comment by kvz** [comment]
It seems to me there may be an issue here:

```js
github.event == 'pull_request'
// versus:
github.event.action == 'opened'
```

I don't think `github.event` will be a string but rather an object

**Comment by kvz** [comment]
New to this, but i _think_ you need `github.event_name == 'pull_request' && github.event.action == 'opened' `

**Comment by srggrs** [comment]
@zigang93 @kvz the new release (1.2.1) should fix this! let me know if not! Fixed by #62
